scripts/jenkins/code-coverage-with-xperior-and-doxygen.groovy

```diff
@@ -0,0 +1,126 @@
+pipeline {
+	agent {
+		node {
+			label 'motr-remote-controller-vm-0593'

```

Comment:
Avoid adding specific VM details in Jenkinsfile. You can create a label and add multiple (at least 2) VMs to it.  e.g. https://eos-jenkins.colo.seagate.com/label/ceph-build-node/

Reply:
Above is the label for one particular VM, since I have only 1 dedicated VM for this hence given label specific to VM.